components/_components.scss

```diff
@@ -0,0 +1,242 @@
+// .fa-twitter {
+//   font-size: 32px;
+// }
+.tweet-btn {
+  width: 100%;
+  display: flex;
+  //   justify-content: center;
+  #tweet-btn {
+    text-align: center;
+    border: transparent;
+    border-radius: 200px;
+    color: #fff;
+    font-size: 2rem;
+    padding: 1.4rem 8rem;
+    background-color: $color-main;
+    &:hover {
+      background-color: $color-main-dark;
+      transition: all 0.3s;
```

Comment:
What does transition do?

Reply:
It let you to change certain styles like color, size, opacity, under a specific time.